### PR TITLE
Add new data to the repo

### PR DESCRIPTION
**Pull request by cw75**
## Describe your changes and why you are making these changes
This PR adds 2 new datasets to the repo. Will make a follow-up PR to insert these tables into the user's demo db.

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/resources/demo_db/sqlite/v0.1.0/create_tables.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+from sqlalchemy import create_engine, engine
+from pathlib import Path
+from os.path import join
+
+def create_diabetes_table(engine):
+    df = pd.read_csv(
+        "diabetes.csv",
```
Comment (cw75) [inline src/resources/demo_db/sqlite/v0.1.0/create_tables.py:8]: This file path will be changed in a follow-up PR.

File: src/resources/demo_db/sqlite/v0.1.0/create_tables.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    df.to_sql(
+        "diabetes",
+        con=engine,
+        index=False,
+        if_exists="replace",
+    )
+
+def create_housing_table(engine):
+    df = pd.read_csv("house_prices.csv", index_col="Id")
```
Comment (cw75) [inline src/resources/demo_db/sqlite/v0.1.0/create_tables.py:31]: This file path will be changed in a follow-up PR.

**Review by hsubbaraj-spiral — APPROVED** [review]

**Comment by likawind** [comment]
what are the version numbers for?

**Review by likawind — APPROVED** [review]

**Comment by cw75** [comment]
@likawind v0.0.1 means the data existed since the very first release, and v0.1 means the data are new as of the v0.1 release.

**Pull request merged**